benefits/static/css/admin/styles.css

```diff
@@ -63,3 +73,19 @@ html[data-theme="light"],
 #logout-form button {
   text-transform: unset;
 }
+
+.checkbox-parent .form-group:last-of-type .col-12 {
+  display: flex;
+  flex-direction: row-reverse;
+  justify-content: start;
+  column-gap: 0.5rem;
+  margin-top: 2rem;
+}
+
+.checkbox-parent,
+.checkbox-parent .form-group .col-12,
+.checkbox-parent .form-group .col-12 #id_flow {
+  display: flex;
+  flex-direction: column;
+  row-gap: 1rem;
+}
```

Comment:
As utility classes: `d-flex flex-row-reverse justify-content-start gap-0 row-gap-1` 

`d-flex flex-column gap-0 row-gap-3` 

But just decided to do it like this for now. Named `checkbox-parent` because the main thing this does is flip the checkbox rendering to show the checkbox input first, then label, which is the opposite of the rest of the pages.